external/elasticsearch/playlist.xml

```diff
@@ -27,7 +27,7 @@
 		</groups>
 	</test>
 	<test>
-		<testCaseName>elasticsearch_test_openj9_jdk8</testCaseName>
+		<testCaseName>elasticsearch_test</testCaseName>
```

Comment:
This can stay as `elasticsearch_test_openj9_jdk8`, we will be changing the elasticsearch targets in a separate PR.